src/neo/VM/Helper.cs

```diff
@@ -219,6 +220,56 @@ public static byte[] MakeScript(this UInt160 scriptHash, string operation, param
             }
         }
 
+        public static JObject ToJson(this StackItem item)
+        {
+            return ToJson(item, null);
+        }
+
+        private static JObject ToJson(StackItem item, HashSet<StackItem> context)
+        {
+            JObject json = new JObject();
+            json["type"] = item.Type;
+            switch (item)
+            {
+                case Array array:
+                    context ??= new HashSet<StackItem>(ReferenceEqualityComparer.Default);
+                    if (!context.Add(array)) throw new InvalidOperationException();
+                    json["value"] = new JArray(array.Select(p => ToJson(p, context)));
+                    break;
+                case Boolean boolean:
+                    json["value"] = boolean.ToBoolean();
+                    break;
+                case Buffer buffer:
+                    json["value"] = Convert.ToBase64String(buffer.InnerBuffer);
+                    break;
+                case ByteString byteString:
+                    json["value"] = Convert.ToBase64String(byteString.Span);
+                    break;
+                case Integer integer:
+                    json["value"] = integer.ToBigInteger().ToString();
+                    break;
+                case Map map:
+                    context ??= new HashSet<StackItem>(ReferenceEqualityComparer.Default);
+                    if (!context.Add(map)) throw new InvalidOperationException();
+                    json["value"] = new JArray(map.Select(p =>
+                    {
+                        JObject item = new JObject();
+                        item["key"] = ToJson(p.Key, context);
+                        item["value"] = ToJson(p.Value, context);
+                        return item;
+                    }));
+                    break;
+                case Pointer pointer:
+                    json["value"] = pointer.Position;
+                    break;
+                case Null _:
+                    json["value"] = JObject.Null;
+                    break;
+                default: throw new InvalidOperationException();
```

Comment:
We want to allow Interop here? 🤔 ?

Reply:
There is no reverse conversion for this function, only output information, so Interop should be allowed.